include/pops/network.hpp

```diff
@@ -172,43 +199,42 @@ class Network
      *
      * ```
      * Network network(...);
-     * std::string node_filename = "nodes.txt";
-     * std::string segment_filename = "segments.txt"
-     * std::ifstream node_stream{node_filename};
-     * std::ifstream segment_stream{segment_filename};
-     * network.load(node_stream, segment_stream);
+     * std::string filename = "network.txt"
+     * std::ifstream stream{filename};
+     * network.load(stream);
      * ```
      *
-     * The input network is a list of nodes and their coordinates and a list of edges
+     * The input network is a list of edges which are pairs of nodes
      * and the corresponding segments. The input is used as-is. The network is agnostic
```

Comment:
@petrasovaa said:

> Should it rather be "list of segments which are pairs of nodes and the corresponding edges"?
Or "where each segments consists of a pair of nodes and a corresponding edge".

Reply:
Addressed in 5963d739fae0e74a9b63403605e8bbeaac97d3af